proto/ibc/lightclients/wasm/v1/wasm.proto

```diff
@@ -30,4 +30,10 @@ message ClientMessage {
   option (gogoproto.goproto_getters) = false;
 
   bytes data = 1;
+}
+
+// CodeHashes defines a list of all code hashes that are stored.
+message CodeHashes {
```

Comment:
this naming throws me off. `CodeHashes.code_hashes`, maybe `CodeHashes.hashes` sounds better, WDYT?